src/mongodb/scores.controller.ts

```diff
@@ -1575,9 +1577,27 @@ export class ScoresController {
         // End Constructed Text Logic
 
         let fluencyScore = await this.scoresService.getCalculatedFluency(textEvalMatrices, repetitions, originalText, responseText, pause_count);
-
         let createdAt = new Date().toISOString().replace('Z', '+00:00')
 
+        // Add check for the correct choice
+
+        if (is_correct_choice !== undefined && is_correct_choice !== null) {
+          if (is_correct_choice) {
+            correct_choice_score = 20
+          }
+
+          // calculation for the correct choice final score 
+          let similarityDenoised = similarityDenoisedText * 100
+          let key_word = CreateLearnerProfileDto.correctness['50%']
+          const allWordsPresent = key_word.every(word => responseText.includes(word.toLowerCase()));
+
+          if (correct_choice_score > 0 && similarityDenoised > 70) {
```

Comment:
Make similarityDenoised > 70 to similarityDenoised >= 70

Reply:
OK